src/components/UserMasterPanel.js

```diff
@@ -87,6 +90,19 @@ const UserMasterPanel = (props) => {
     handleEmailChange(edited?.email);
   }, []);
 
+  const [showPassword, setShowPassword] = React.useState(false);
```

Comment:
If you're importing 'useState' from 'react', there is not need to use this 'React.' before it.